packages/shared/lib/core/nfts/interfaces/nft.interface.ts

```diff
@@ -6,6 +6,6 @@ export interface INft {
     name: string
     issuer: AddressTypes
     metadata: string
-    parsedMetadata?: IIrc27Metadata
+    parsedMetadata?: IIrc27Metadata | string
```

Comment:
why can this be a string? if `metadata` is not json-parsable then `parsedMetadata` is undefined